packages/blocks/src/lib/slam-map-2d/color-map.ts

```diff
@@ -45,9 +45,19 @@ const colorBuckets = (probability: number): THREE.Vector3 => {
 export const mapColorAttributeGrayscale = (colors: THREE.BufferAttribute) => {
   for (let i = 0; i < colors.count; i += 1) {
     /*
-     * Probability is currently assumed to be held in the rgb field of the PCD map, on a scale of 0 to 100.
-     * ticket to look into this further https://viam.atlassian.net/browse/RSDK-2605
+     * Probability is currently assumed to be held in the b part of the rgb field of the PCD map, on a scale of 0 to 100.
+     * ticket to look into this further https://viam.atlassian.net/browse/RSDK-2605.
+     *
+     * User generated points that have been added to the pcd are marked as such by putting 100% probability
+     * in the r part of the rgb field. If that has been set, we will draw the point in blue instead of mapping it to its
+     * corresponding shade of gray to signal it is a user generated point.
      */
+
+    if (colors.getX(i) === 1) {
+      colors.setXYZ(i, 0, 0, 1);
```

Comment:
Could we color this something other than blue? (Apologies, I see there was already some discourse about the colors.)

Generally, we have been thinking about our design language in visualization tools and color is often used to indicate meaning. It is quite standard for x, y, and z to be displayed as red, green, and blue. Although this is a 2D map, blue still has meaning in our other visualization tools.

Would you be open to picking a different color? I'm happy to brainstorm further, and we could even grab Nat for a few minutes as well.

Here's a list of our brand colors, which might have a good choice: https://github.com/viamrobotics/prime/blob/main/packages/core/theme.ts#L65-L75

And you can import them like so:

```
import { theme } from '@viamrobotics/prime-core/theme';
```

Then you could make a file-level constant for your chosen color:

```
const USER_GENERATED_POINT_COLOR = new THREE.Color(theme.extend.colors.cyberpunk);
```

And use it to set the attribute:

```
colors.setXYZ(i, USER_GENERATED_POINT_COLOR.r, USER_GENERATED_POINT_COLOR.g, USER_GENERATED_POINT_COLOR.b);
```

Reply:
No problem, will update the color. @EshaMaharishi do you have a preference on the color?

Reply:
'raspberry' looks good to me, but happy to go with something else if Ethan/Nat prefer!

Reply:
going with cyberpunk as per Nat's request